fastai2/vision/utils.py

```diff
@@ -44,4 +44,35 @@ def verify_image(fn):
 # Cell
 def verify_images(fns):
     "Find images in `fns` that can't be opened"
-    return L(fns[i] for i,o in enumerate(parallel(verify_image, fns)) if not o)
\ No newline at end of file
+    return L(fns[i] for i,o in enumerate(parallel(verify_image, fns)) if not o)
+
+# Cell
+def resize_image(file, max_size:None, dest=None, n_channels=3, ext=None,
+                 img_format=None, resume=False, **kwargs ):
+    verify_image(file)
+    img = Image.open(file)
+    imgarr = np.array(img)
+    img_channels = 1 if len(imgarr.shape) == 2 else imgarr.shape[2]
+    if (max_size is not None and (img.height > max_size or img.width > max_size)) or img_channels != n_channels:
+        assert isinstance(dest, Path), "You should provide `dest` Path to save resized image"
```

Comment:
One dest is a required argument, we can remove this line, and add a dest=Path(dest) at the beginning to be sure it's a Path.